src/main/kotlin/io/sc3/goodies/Registration.kt

```diff
@@ -414,6 +426,12 @@ object Registration {
     fun <T : Item> rItem(name: String, value: T): T =
       register(ITEM, ModId(name), value).also { items.add(it) }
 
+    private fun rIceCreamItem(name: String) : Item
+    {
```

Comment:
Should be `private fun rIceCreamItem(name: String): Item {`, but since this immediately just returns you could also just do,

```kotlin
private fun rIceCreamItem(name: String): Item = rItem(name, IceCreamItem(itemSettings()
  .food(IceCreamItem.foodComponent)
  .maxCount(16)))
```

which would be preferable.